content/en/docs/topics/advanced.md

```diff
@@ -80,6 +80,97 @@ type PostRenderer interface {
 
 For more information on using the Go SDK, See the [Go SDK section](#go-sdk)
 
+#### Simple Example
+
+Here is `custom.go`
```

Comment:
can we change this sentence to be more descriptive.? Something like the following: This is a simple example of a custom post renderer which reads the Helm rendered manifests, adds a label to `deployment` and then outputs the modified manifests.